### PR TITLE
Spanish translation of  static-site-generation.md

### PR DESCRIPTION
**Pull request by d4g0**

### CODE REVIEW

**Comment by debs-obrien** [comment]
@Kronhyx are you available to give this a quick review please

File: static-site-generation.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+</base-alert>
+
+### Actualizando tu contenido
+
+Con el fin de obtener nuevo contenido desde tu API solo es necesario regenerar tu sitio nuevamente. En la mayoría de los proveedores de hosting estático puedes hacer esto con un push con los cambios al master branch mediante el comando de git o haciendo un pull request.
+
+### Modo de vista previa
+
+Modo de vista previa llamará a tu API o tu CMS para que veas los cambios antes de desplegar. Ver el [preview mode](/guides/features/live-preview) on how to enable this feature.
```
Comment (debs-obrien) [inline static-site-generation.md:72]: missing translation for this line

**Comment by alvarosaburido** [comment]
@debs-obrien isn't it changing the same file as in #772

File: static-site-generation.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- True
+- False
+correctAnswer: False
+- question: Qué comando usas para generar tu sitio estático ?
+answers:
+- nuxt build
+- nuxt prerender
+- nuxt generate
+correctAnswer: nuxt generate
+- question: Cuando es llamada tu API ?
```
Comment (Kronhyx) [inline static-site-generation.md:18]:
```suggestion
- question: Cuándo es llamada tu API ?
```

File: static-site-generation.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Cuando generas tu sitio
+- Cuando generas tu sitio y cada vez que navegas a la página con el contenido del API
+correctAnswer: Cuando generas tu sitio
+- question: Que páginas alternarán a single page application mode ?
+answers:
+- La página de error
+- Las páginas excluidas del proceso de generación con generate.excludes 
+- Todas páginas en navegación
+correctAnswer: Las páginas excluidas del proceso de generación con generate.excludes
+- question: Como actualizas el contenido de tu sitio ?
```
Comment (Kronhyx) [inline static-site-generation.md:30]:
```suggestion
- question: Cómo actualizas el contenido de tu sitio ?
```

**Review by Kronhyx — CHANGES_REQUESTED** [review]
Just spelling mistakes

File: static-site-generation.md (hunk 1 above)
Comment (Kronhyx) [reply static-site-generation.md:72]:
```suggestion
Modo de vista previa llamará a tu API o tu CMS para que veas los cambios antes de desplegar. Ver el [preview mode](/guides/features/live-preview) sobre como activar esta funcionalidad.
```

**Comment by debs-obrien** [comment]
hey @alvarosaburido welcome back. Hope you had a good birthday and enjoyed the wine. Yes it seems to be the last file and has been translated twice so one of the pr's needs to be merged and I have no idea which one to merge. I think https://github.com/nuxt/nuxtjs.org/pull/772 has some issues to be fixed and was thinking that perhaps we could merge this one and perhaps @FelixMor could review it and add any of the changes he has in his that he thinks are better. what do you think Alvaro?

**Comment by alvarosaburido** [comment]
> hey @alvarosaburido welcome back. Hope you had a good birthday and enjoyed the wine. Yes it seems to be the last file and has been translated twice so one of the pr's needs to be merged and I have no idea which one to merge. I think #772 has some issues to be fixed and was thinking that perhaps we could merge this one and perhaps @FelixMor could review it and add any of the changes he has in his that he thinks are better. what do you think Alvaro?

hehe thanks 😋, yes,  I agree, @FelixMor  let me know if you need a hand.

**Pull request merged**